### PR TITLE
Update filter documentation

### PR DESCRIPTION
**Pull request by ketankartoza**
This PR is for issue #1640

### CODE REVIEW

**Comment by ketankartoza** [comment]
@Jeremy-Prior Please review.

File: docs/src/user/manual/explore/filters.md [hunk 1]
```diff
@@ -18,19 +18,38 @@ The filter side panel enhances the user experience by allowing them to focus on
 
 #### 1. Search place Filter
 
-* **Filter Description:** This filter allows users to search place.
-* **Usage:** Click on search place to search the places.
+* **Filter Description:** This filter allows users to search places.
```
Comment (Jeremy-Prior) [inline docs/src/user/manual/explore/filters.md:21]:
```suggestion
* **Filter Description:** This filter allows users to search for places.
```

File: docs/src/user/manual/explore/filters.md [hunk 2]
```diff
@@ -18,19 +18,38 @@ The filter side panel enhances the user experience by allowing them to focus on
...
+* **Usage:** Click on the search place to search the places.
```
Comment (Jeremy-Prior) [inline docs/src/user/manual/explore/filters.md:22]:
```suggestion
* **Usage:** Click on the search place field and type letters to search for places.
```

File: docs/src/user/manual/explore/filters.md [hunk 3]
```diff
@@ -18,19 +18,38 @@ The filter side panel enhances the user experience by allowing them to focus on
...
 
 ![Search_place filter](./img/filters-3.png)
 
-1. **Search place:** Click on search place to search the places and filter.
+1. **Search place:** Click on the search place to search the places and filter.
```
Comment (Jeremy-Prior) [inline docs/src/user/manual/explore/filters.md:26]:
```suggestion
1. **Search place:** Click on the search place to filter places.
```

File: docs/src/user/manual/explore/filters.md [hunk 4]
```diff
@@ -39,61 +58,34 @@ to make user group e.g. `National data scientist` able to see the organisation f
...
+#### 5. Property Filter
 
-* **Filter Description:** The property filter helps users filter data based on properties. This filter is dependent on Organisation Filter. 
-It means, the options will be based on the selected Organisations.
+* **Filter Description:** The property filter helps users filter data based on properties. This filter is dependent on the organisation filter. 
+It means the options will be based on the selected organisations.
 * **Usage:** Select a property from the available options to view data related to that property.
-* **Permission:** `Can view property filter` permission must be added to the Group on the admin page, 
-to make user group e.g. `National data scientist` able to see the property filter.
+* **Permission:** `Can view property filter` permission must be added to the group on the admin page,
```
Comment (Jeremy-Prior) [inline docs/src/user/manual/explore/filters.md:68]:
```suggestion
* **Permission:** The `can view property filter` permission must be added to the group on the admin page, 
```

File: docs/src/user/manual/explore/filters.md [hunk 5]
```diff
@@ -39,61 +58,34 @@ to make user group e.g. `National data scientist` able to see the organisation f
...
 
-* **Filter Description:** The property filter helps users filter data based on properties. This filter is dependent on Organisation Filter. 
-It means, the options will be based on the selected Organisations.
+* **Filter Description:** The property filter helps users filter data based on properties. This filter is dependent on the organisation filter. 
+It means the options will be based on the selected organisations.
 * **Usage:** Select a property from the available options to view data related to that property.
-* **Permission:** `Can view property filter` permission must be added to the Group on the admin page, 
-to make user group e.g. `National data scientist` able to see the property filter.
+* **Permission:** `Can view property filter` permission must be added to the group on the admin page, 
+to make the user group e.g. `National data scientist` able to see the property filter.
```
Comment (Jeremy-Prior) [inline docs/src/user/manual/explore/filters.md:69]:
```suggestion
to allow the user group (e.g. `National data scientist`) to see the property filter.
```

File: docs/src/user/manual/explore/filters.md [hunk 6]
```diff
@@ -39,61 +58,34 @@ to make user group e.g. `National data scientist` able to see the organisation f
...
 
 ![Property filter](./img/filters-6.png)
 
-1. **Select all:** Select all selects all the properties.
+1. **Select All:** `Select All` select all the properties.
```
Comment (Jeremy-Prior) [inline docs/src/user/manual/explore/filters.md:73]:
```suggestion
1. **Select All:** `Select All` of the properties.
```

File: docs/src/user/manual/explore/filters.md [hunk 7]
```diff
@@ -103,6 +95,15 @@ It means, the options will be based on the selected Organisations.
 1. **Critical biodiversity type dropdown** Using this, the user can filter the data according to the critical biodiversity areas.
 2. **Ecosystem dropdown:** Using this, the user can filter the data according to the ecosystem types.
 
+8. **Clear All**: Click on the `Clear All` button, to clear all the filters.
+#### Report Type Filter
```
Comment (Jeremy-Prior) [inline docs/src/user/manual/explore/filters.md:99]:
```suggestion
8. **Clear All**: Click on the `Clear All` button to clear all the filters.

#### Report Type Filter
```

File: docs/src/user/manual/explore/filters.md [hunk 8]
```diff
@@ -103,6 +95,15 @@ It means, the options will be based on the selected Organisations.
...
+
+* **Filter Description:** This filter allows users to filter data based on report types. This filter appears on the reports tab. 
+* **Usage:** Choose a report type from the list to view data associated with that report type.
+
+![Report filter](./img/filters-5.png)
+
+1. **Report type:** The Report Type Filter is a multi-select filter that enables users to filter data based on specific report types or view data of all the report types. With this filter, users can easily narrow down their dataset to view data associated with a particular report type.
 ## Summary
```
Comment (Jeremy-Prior) [inline docs/src/user/manual/explore/filters.md:107]:
```suggestion
1. **Report type:** The Report Type Filter is a multi-select filter that enables users to filter data based on specific report types or view data of all the report types. With this filter, users can easily narrow down their dataset to view data associated with a particular report type.

## Summary
```

File: docs/src/user/manual/explore/filters.md [hunk 9]
```diff
@@ -103,6 +95,15 @@ It means, the options will be based on the selected Organisations.
...
 
-The Data Table Page is a comprehensive platform for exploring, analysing, and interacting with the user's data. With filters, column search, and data export options, the user has the tools necessary to customise their data view, find specific information, and export data in the format that suits their needs.
+The filter side panel provides a robust toolset for users to refine and explore their datasets effectively. Whether it's searching for specific places, species, or filtering by activities, organisations, properties, years, or spatial criteria, this comprehensive guide ensures users can seamlessly navigate and utilize the filtering functionalities for a more tailored data analysis experience.
```
Comment (Jeremy-Prior) [inline docs/src/user/manual/explore/filters.md:109]:
```suggestion
The filter side panel provides a robust toolset for users to refine and explore their datasets effectively. Whether it's searching for specific places, species, or filtering by activities, organisations, properties, years, or spatial criteria, this comprehensive manual ensures users can seamlessly navigate and utilize the filtering functionalities for a more tailored data analysis experience.
```

**Review by Jeremy-Prior — CHANGES_REQUESTED** [review]

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 1 min 32 sec

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 1 min 47 sec

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 1 min 32 sec

**Comment by ketankartoza** [comment]
@Jeremy-Prior Please review.

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 2 min 5 sec

File: docs/src/user/manual/explore/filters.md [hunk 10]
```diff
@@ -103,6 +95,17 @@ It means, the options will be based on the selected Organisations.
...
+* **Usage:** Choose a report type from the list to view data associated with that report type.
+
+![Report filter](./img/filters-5.png)
+
+1. **Report type:** The Report Type Filter is a multi-select filter that enables users to filter data based on specific report types or view data of all the report types. With this filter, users can easily narrow down their dataset to view data associated with a particular report type.
+
 ## Summary
 
-The Data Table Page is a comprehensive platform for exploring, analysing, and interacting with the user's data. With filters, column search, and data export options, the user has the tools necessary to customise their data view, find specific information, and export data in the format that suits their needs.
+The filter side panel provides a robust toolset for users to refine and explore their datasets effectively. Whether it's searching for specific places, species, or filtering by activities, organisations, properties, years, or spatial criteria, this comprehensive manual ensures users can seamlessly navigate and utilize the filtering functionalities for a more tailored data analysis experience.
```
Comment (Jeremy-Prior) [inline docs/src/user/manual/explore/filters.md:111]:
```suggestion
The filter side panel provides a robust toolset for users to refine and explore their datasets effectively. Whether it's searching for specific places, species, or filtering by activities, organisations, properties, years, or spatial criteria, this comprehensive manual ensures users can seamlessly navigate and utilise the filtering functionalities for a more tailored data analysis experience.
```

**Review by Jeremy-Prior — CHANGES_REQUESTED** [review]

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 2 min 23 sec

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 2 min 6 sec

**Pull request merged**